src/util/wallet-utils/src/checkWallets.ts

```diff
@@ -21,5 +22,6 @@ export const checkAvailableWallets: () => Promise<{
   return {
     leather: isLeather,
     xverse: isXverse,
+    asigna: window.top !== window.self,
```

Comment:
this check is waaay too broad can we have a checker in the asigna connect library that might include like a checker for the host maybe 

<img width="233" alt="image" src="https://github.com/user-attachments/assets/68e8e05d-09c6-4c41-973a-f960a6c5b23b" />

That's the only blocker for now